specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-07-01-preview/elastic.json

```diff
@@ -0,0 +1,2763 @@
+      "description": "Marketplace Subscription and Organization details to which resource gets billed into.",
+      "type": "object",
+      "properties": {
+        "marketplaceSaasInfo": {
+          "description": "Marketplace Subscription details",
+          "$ref": "#/definitions/MarketplaceSaaSInfo"
+        },
+        "partnerBillingEntity": {
+          "description": "Partner Billing Entity details: Organization Info",
+          "$ref": "#/definitions/PartnerBillingEntity"
+        }
+      }
+    },
+    "MarketplaceSaaSInfo": {
+      "description": "Marketplace SAAS Info of the resource.",
+      "type": "object",
+      "properties": {
+        "marketplaceSubscription": {
+          "description": "Marketplace Subscription",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "Marketplace Subscription Id. This is a GUID-formatted string.",
+              "type": "string"
+            }
+          }
+        },
+        "marketplaceName": {
+          "description": "Marketplace Subscription Details: SAAS Name",
+          "type": "string"
+        },
+        "marketplaceResourceId": {
+          "description": "Marketplace Subscription Details: Resource URI",
+          "type": "string"
+        },
+        "marketplaceStatus": {
+          "description": "Marketplace Subscription Details: SaaS Subscription Status",
+          "type": "string"
+        },
+        "billedAzureSubscriptionId": {
+          "description": "The Azure Subscription ID to which the Marketplace Subscription belongs and gets billed into.",
+          "type": "string"
+        },
+        "subscribed": {
+          "description": "Flag specifying if the Marketplace status is subscribed or not.",
+          "type": "boolean"
+        }
+      }
+    },
+    "PartnerBillingEntity": {
+      "description": "Partner Billing details associated with the resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The Elastic Organization Id.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The Elastic Organization Name.",
+          "type": "string"
+        },
+        "partnerEntityUri": {
+          "description": "Link to the elastic organization page",
+          "type": "string"
+        }
+      }
+    },
+    "ConnectedPartnerResourceProperties": {
+      "description": "Connected Partner Resource Properties",
+      "type": "object",
+      "properties": {
+        "partnerDeploymentName": {
+          "description": "Elastic deployment name",
+          "type": "string"
+        },
+        "partnerDeploymentUri": {
+          "description": "Deployment URL of the elasticsearch in Elastic cloud deployment.",
+          "type": "string"
+        },
+        "liftrResourceId": {
+          "description": "The azure resource Id of the deployment.",
+          "type": "string"
+        },
+        "location": {
```

Comment:
>location [](http://example.com/codeflow?start=9&length=8)

Is this an ARM region, or something else?

Reply:
ARM region where resource is deployed. I want to send list of all the resources created under one organization with their region. Therefore, it is arm region only.